docker/sql/sandbox.sql

```diff
@@ -0,0 +1,70 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 10.19 (Ubuntu 10.19-0ubuntu0.18.04.1)
+-- Dumped by pg_dump version 10.19 (Ubuntu 10.19-0ubuntu0.18.04.1)
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+--
+-- Name: DATABASE sandbox; Type: COMMENT; Schema: -; Owner: dev_epigraph
+--
+
+COMMENT ON DATABASE sandbox IS 'Database for tests and experimentation. Data saved here can be deleted at any time.';
+
+
+--
+-- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: 
+--
+
+CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
+
+
+--
+-- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: 
+--
+
+COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';
+
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: test_maroussia; Type: TABLE; Schema: public; Owner: dev_epigraph
+--
+
+CREATE TABLE public.test_maroussia (
```

Comment:
On the sandbox database, we can re-create it empty, and not with the previously existing tables. The Idea of the sandbox is to be used for non-persistent experimentation. In this respect we can even add a CRON script to drop all of its tables every month or so...

Reply:
oh I see .. thanks for the information! I will update that!